app/src/lib/pocket.ts

```diff
@@ -0,0 +1,56 @@
+import { Configuration, HttpRpcProvider, Pocket, PocketRpcProvider, QueryAppResponse, RpcError, typeGuard } from "@pokt-network/pocket-js"
+import { getAddressFromPublicKey } from '../utils/conversions';
+
+const blockTime = process.env.BLOCK_TIME
+
+const DEFAULT_DISPATCHER_LIST = 'https://peer-1.nodes.pokt.network:4201'
```

Comment:
Just a bit of a reminder, but as we pinpoint the "yeeting dispatchers" bug we're facing on the portal API we probably want to replicate the solution here.